packages/apex-node/src/tests/asyncTests.ts

```diff
@@ -295,23 +295,15 @@ export class AsyncTests {
     apexTestResultQuery += 'FROM ApexTestResult WHERE QueueItemId IN (%s)';
 
     const apexResultIds = testQueueResult.records.map(record => record.Id);
-    let formattedIds = '';
-    const queries = [];
 
     // iterate thru ids, create query with id, & compare query length to char limit
-    for (const id of apexResultIds) {
-      const newIds = addIdToQuery(formattedIds, id);
-      const query = util.format(apexTestResultQuery, `'${newIds}'`);
-
-      if (query.length > QUERY_CHAR_LIMIT) {
-        queries.push(util.format(apexTestResultQuery, `'${formattedIds}'`));
-        formattedIds = '';
-      }
-      formattedIds = addIdToQuery(formattedIds, id);
```

Comment:
Good work on rewriting this part using array operations, should be faster than `addIdToQuery`. If `addIdToQuery` is no longer used can it be removed?

Reply:
Good catch, I just removed `addIdToQuery()` from utils.ts